spec/integration/aws_cloudsearch_domain_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe Chef::Resource::AwsCloudsearchDomain do
+  extend AWSSupport
+  when_the_chef_12_server "exists", organization: "foo", server_scope: :context do
+    with_aws "when connected to AWS" do
+      it "aws_cloudsearch_domain 'test-cloudsearch-domain' creates a cloudsearch domain" do
```

Comment:
This test isn't too slow.  The comment about slowness was with adding additional assertions since some data doesn't show up in the API for a *very* long time (on the order of 10s of minutes).